<p>Erro durante a restauração da instância <strong>{{ $instance_name }}</strong> do backup. Mensagem: {{ $exception_message }}
</p>
